Anime-App/Feature/Anime/View/SettingsViewController.swift

```diff
@@ -0,0 +1,179 @@
+        let audioLbl = UILabel()
+        audioLbl.translatesAutoresizingMaskIntoConstraints = false
+        audioLbl.textColor = .black
+        audioLbl.text =  "Audio"
+        audioLbl.font = .init(name: "", size: 14)
+        return audioLbl
+    }()
+    
+    private lazy var hapticFeedbackSwitch: UISwitch = {
+        let hapticFeedbackSwitch = UISwitch()
+        hapticFeedbackSwitch.translatesAutoresizingMaskIntoConstraints = false
+        hapticFeedbackSwitch.isOn = false
+        hapticFeedbackSwitch.addTarget(self, action: #selector(toggleHapticFeedback), for: .touchUpInside)
+        return hapticFeedbackSwitch
+    }()
+    
+    private lazy var audioSwitch: UISwitch = {
+        let audioSwitch = UISwitch()
+        audioSwitch.addTarget(self, action: #selector(toggleAudio), for: .touchUpInside)
+        audioSwitch.translatesAutoresizingMaskIntoConstraints = false
+        return audioSwitch
+    }()
+    
+    @objc func toggleHapticFeedback() {
+        if hapticFeedbackSwitch.isOn == true {
+            print("haptic Feedback is on")
+            defaults.set(true, forKey: "UseHaptic")
+        } else {
+            UserDefaults.standard.set(false, forKey: "UseHaptic")
+            guard let appDomain = Bundle.main.bundleIdentifier else { return }
+            UserDefaults.standard.removePersistentDomain(forName: appDomain)
+            UserDefaults.standard.removeObject(forKey: "UseHaptic")
+        }
+    }
+    
+    @objc func toggleAudio() {
+        if audioSwitch.isOn == true {
+            let audioIsOn = audioSwitch.isOn == true
+            print("Audio is on")
+            defaults.set(audioIsOn, forKey: "UseAudio")
+        } else {
+            guard let appDomain = Bundle.main.bundleIdentifier else { return }
+            UserDefaults.standard.removePersistentDomain(forName: appDomain)
+            UserDefaults.standard.removeObject(forKey: "UseAudio")
+        }
+    }
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setup()
+     }
+
+
+}
+
+extension SettingsViewController {
+    func setup() {
+        view.backgroundColor = .white
+        view.addSubview(stackView)
+        let textAttributes = [NSAttributedString.Key.foregroundColor:UIColor.black]
+        navigationController?.navigationBar.titleTextAttributes = textAttributes
+        self.title = "Setting"
+        
+        stackView.addArrangedSubview(animeImageView)
+        stackView.addArrangedSubview(appVersionLbl)
+
+        stackView.addArrangedSubview(hapticFeedBackStackView)
+        hapticFeedBackStackView.addArrangedSubview(hapticFeedbackLbl)
+        hapticFeedBackStackView.addArrangedSubview(hapticFeedbackSwitch)
+        
+        stackView.addArrangedSubview(audioStackView)
+        audioStackView.addArrangedSubview(audioLbl)
+        audioStackView.addArrangedSubview(audioSwitch)
+        
+        
+        if traitCollection.userInterfaceIdiom == .mac {
```

Comment:
I don't get why you're doing this? Are you build the app for mac? If you copy code can you just start to think. If you need to actually use the code you're copying. Rather than just adding it in. Reason why is that this looks really bad when you do this in a job or in a tech test cos it shows and means you don't know what is going on. This is massive red flag to employers